src/defaultUserImage.ts

```diff
@@ -28,3 +33,29 @@ export const getDefaultUserImage = async (): Promise<ImageData> => {
         throw new Error('unsupported platform: ' + Platform.OS);
     }
 };
+
+export const copyImageToApp = async (image: ImageData, filename: string): Promise<string> => {
+    if (image.data != null) {
+        const path = await getPathForFile(filename);
+        if (!await RNFS.exists(path)) {
+            try {
+                await RNFS.writeFile(path, image.data, 'base64');
```

Comment:
This doesn't seem to be good, the data url header should be stripped

Reply:
it is not there